qdrant/resource_accounts_cluster_test.go

```diff
@@ -121,3 +121,79 @@ output "cluster_cfg_num_nodes" {
 		testCase(t, "apply")
 	})
 }
+
+func TestResourceClusterCreateWithExtraDisk(t *testing.T) {
+	provider := fmt.Sprintf(`
+provider "qdrant-cloud" {
+  api_key = "%s"
+}
+	`, os.Getenv("QDRANT_CLOUD_API_KEY"))
+
+	config := provider + fmt.Sprintf(`
+data "qdrant-cloud_booking_packages" "test" {
+  cloud_provider = "gcp"
+  cloud_region   = "us-east4"
+}
+locals {
+  resource_data = data.qdrant-cloud_booking_packages.test.packages
+  // Filter out the gpx1
```

Comment:
filter out is double imo: Filter out the gpx1 --> Filter on gpx1

Reply:
Fixed [here](https://github.com/qdrant/terraform-provider-qdrant-cloud/commit/0fda97fd6f3b8b48377a79a85ce23ca2d4909e74)